### PR TITLE
60-edit-training-program :: Edit training program

### PR DESCRIPTION
**Pull request by daynewright**
## Status
**READY**

## Documentation

[X] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[ ] This PR does **NOT** require documentation.

## Description
Creates edit view and submits edited training program.

## Resolves Issue Number
#60 

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

```sh
git pull --prune
git checkout 60-edit-training-program
dotnet run
```

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.

1.  dotnet run
2.  navigate to training programs
3.  click on a training program
4.  click edit
5.  edit it and save
6.  enjoy

## Impacted Files in Application
List general components of the application that this PR will affect:

* TrainingProgramController.cs
* TrainingProgramEdit.cs
* Edit.cshtml

### CODE REVIEW

File: src/workforce-management/Controllers/TrainingProgramController.cs [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
+using System.Threading.Tasks;
+using System;
 
 namespace workforce_management.Controllers
 {
-    public class TrainingProgramController: Controller
+    public class TrainingProgramController : Controller
     {
         /**
          * CLASS: ProductTypes
```
Comment (matthamil) [inline src/workforce-management/Controllers/TrainingProgramController.cs]: "CLASS: ProductTypes"
🤔

File: src/workforce-management/Controllers/TrainingProgramController.cs [hunk 2]
```diff
@@ -1,20 +1,21 @@
...
          * PURPOSE: Creates routes for main index view (buy method) and seller view (sell method)
-         * AUTHOR: Matt Kraatz
+         * AUTHOR: Matt Kraatz/Dayne Wright
          * METHODS:
          *   TrainingProgramController(BangazonContext) - Constructor that saves the database context to a private variable.
          *   IActionResult Index() - Returns a View listing all Training Programs.
```
Comment (matthamil) [inline src/workforce-management/Controllers/TrainingProgramController.cs:17]: Missing documentation for other methods

**Review by matthamil — CHANGES_REQUESTED** [review]
Documentation issues

File: src/workforce-management/Controllers/TrainingProgramController.cs (hunk 1 above)
Comment (daynewright) [reply src/workforce-management/Controllers/TrainingProgramController.cs]: What the what?!? 👀

File: src/workforce-management/Controllers/TrainingProgramController.cs (hunk 2 above)
Comment (daynewright) [reply src/workforce-management/Controllers/TrainingProgramController.cs:17]: ✅

**Comment by daynewright** [comment]
@matthamil Documentation updated.

**Comment by daynewright** [comment]
I fixed issue #74 within this pull request as well.  Be sure to check it too!  😨

**Review by matthamil — APPROVED** [review]

**Comment by matthamil** [comment]
Works like a charm :star2:

**Comment by MattKraatz** [comment]
Looks GREAT to me

**Comment by garrett-vangilder** [comment]
LGTM 👍  💯

**Comment by garrett-vangilder** [comment]
LGTM 👍  💯

**Pull request merged**